cryptorand/cipher.go

```diff
@@ -0,0 +1,48 @@
+package cryptorand
```

Comment:
IMO this doesn't belong in cryptorand package because it's not random, it should go in the `dbcrypt` package or it's own package.

Reply:
Agreed. Will move